test/MerklePatriciaProof.js

```diff
@@ -0,0 +1,118 @@
+const MerklePatriciaProofMock = artifacts.require("./MerklePatriciaProofMock.sol");
+const accountProofJson =  require("./data/AccountProof.json");
+const storageProofJson =  require("./data/StorageProof.json");
+const ethutil = require ("ethereumjs-util");
+
```

Comment:
JS syntax we follow is:
const MerklePatriciaProofMock = artifacts.require("./MerklePatriciaProofMock.sol")
, accountProofJson =  require("./data/AccountProof.json")
, storageProofJson =  require("./data/StorageProof.json")
, ethutil = require ("ethereumjs-util")
;